README.md

```diff
@@ -31,6 +35,62 @@ Alternatively, when using from Lagom:
 libraryDependencies += "com.lightbend" %% "lagom-service-locator-dns" % "0.1.0"
 ```
 
-## Motivation
+Your platform will require differing "transformers" that will translate a service name
+into a DNS SRV name to be looked up. Here is the Typesafe config declaration to be 
+considered:
+
+```
+service-locator-dns {
```

Comment:
The transformers will depend on the target env. It'd be great to have a sample for each, but I'm going to stay focused on dcos for now.